### DISCUSSION TITLE
Activation of Copilot License For Verified Students

### DISCUSSION THREAD

**Discussion by luyandadhlamini2** [community/community#109705] [GitHub Education] [answered]
### Select Topic Area

General

### Body

I was validated as a student on my github account 4 days ago.

I'm visiting my copilot account in Account Settings under Code, planning, and automation, and select Copilot,
I do not  see "You're eligible to use copilot for free" and then a button to claim it. as instructed [here](https://education.github.com/pack/offers#github-codespaces).

See screenshot of my account in Github Education as proof of active membership to Github Global Campus / student verification

![Screenshot 2024-02-23 174507](https://github.com/community/community/assets/160016656/776a1ec0-8c25-4757-9f97-0198909225fd)

1. How do I contact Copilot support (via email/ otherwise).
2. How do I confirm that I'm active as a student (I can successfully login via [Github Education](https://education.github.com/globalcampus/student))?

Thank you

**Comment by fraineralex** [comment, marked as answer]
Hi @luyandadhlamini2 ,

It's not uncommon for the validation process to take some time. In my case, it took over a week. I recommend waiting a bit longer.

Yes, if you can successfully log in via GitHub Education, it means your student account is active.

**Reply by sarwar-asik** [reply]
Same issue with me, still 6 days waiting

**Comment by JohannaPeanut** [comment]
Hello. I've been verified as a student more than 20 days ago - I can see this in my GH profile: "Education Benefits
You have a current student coupon applied." I now want to use Copilot Pro as described [here](https://github.com/education/students). Unfortunately, the option to use Copilot Pro doesn't appear in my profile --> Your Copilot. According to [these instructions](https://docs.github.com/en/copilot/managing-copilot/managing-copilot-as-an-individual-subscriber/getting-started-with-copilot-on-your-personal-account/getting-free-access-to-copilot-pro-as-a-student-teacher-or-maintainer), it should be possible to activate it there. What can I do?
![grafik](https://github.com/user-attachments/assets/0d1015e3-9943-4a7f-8a67-6bbc3e22b634)
